ioservice/io_foms.c

```diff
@@ -861,15 +864,32 @@ static void stobio_complete_cb(struct m0_fom_callback *cb)
 	struct m0_fom           *fom   = cb->fc_fom;
 	struct m0_io_fom_cob_rw *fom_obj;
 	struct m0_stob_io_desc  *stio_desc;
+	struct m0_stob_io       *siod_stob_io;
 
 	M0_PRE(m0_fom_group_is_locked(fom));
 	M0_ASSERT(m0_fom_phase(fom) == M0_FOPH_IO_STOB_WAIT);
 	stio_desc = container_of(cb, struct m0_stob_io_desc, siod_fcb);
 	M0_ASSERT(m0_stob_io_desc_invariant(stio_desc));
+	siod_stob_io = &stio_desc->siod_stob_io;
```

Comment:
This variable is added so that some lines wrap with 80 chars. Is it okay ? Should I remove this and let the line go beyong 80 chars ?